hadoop-hdfs-project/hadoop-hdfs-nfs/src/test/java/org/apache/hadoop/hdfs/nfs/nfs3/TestDFSClientCache.java

```diff
@@ -31,8 +31,14 @@
 import org.apache.hadoop.hdfs.nfs.conf.NfsConfiguration;
 import org.apache.hadoop.security.UserGroupInformation;
 import org.junit.Test;
+import org.junit.AfterClass;
 
 public class TestDFSClientCache {
+  @AfterClass
```

Comment:
nit: maybe @After? Basically reset/clean all side-effects after each test.

Reply:
Agreed, I think that would be better! Just updated the change, thank you.